homeassistant/components/modbus/__init__.py

```diff
@@ -110,6 +111,9 @@
     struct_validator,
 )
 
+_LOGGER = logging.getLogger(__name__)
```

Comment:
Although I haven't done it myself yet, I see many code improvements moving this to `const.py` as `LOGGER` and than just importing it when needed.

Reply:
That works, but then you do not have different names for different parts of the code.